tower_cli/cli/transfer/logging_command.py

```diff
@@ -24,7 +25,9 @@ def print_header_row(self, asset_type, asset_name):
 
         # The 4 is from 2 spaces and 2 brackets
         stars = '*' * (int(self.columns) - (len(asset_type + asset_name) + 4))
-        self.my_print("{} [{}] {}".format(asset_type.replace("_", " ").upper(), asset_name, stars))
+        self.my_print(six.text_type("{} [{}] {}").format(
```

Comment:
Should we do this here or could we do this down at line 95 to cover all "my_print" calls? 
(line 96):
    message = six.text_type(message)

Reply:
I think we have to do it here, because it's the computation of `"{} [{}] {}".format(asset_type.replace("_", " ").upper(), asset_name, stars)` that causes the problem. There are quite a few other cases of using `.format` in this file, but I looked at all of them, and I don't see any other cases where non-ascii characters are likely to be in the arguments to it. If we pass an internal string, like type, then it's not an issue, since it's not user-defined, so it will never contain unicode.

Reply:
Cool, I'm good then.